src/commands/registerApiSubCommands/registerStepByStep.ts

```diff
@@ -17,7 +17,7 @@ export async function registerStepByStep(context: IActionContext, node?: ApisTre
         const apiCenterNode = await ext.treeDataProvider.showTreeItemPicker<ApiCenterTreeItem>(ApiCenterTreeItem.contextValue, context);

```

Comment:
I rememeber `showTreeItemPicker` could pass the param with a start tree item, so with that, end user would not be able to select a DataPlane tree item.

Reply:
the tree view from ext.treeDataProvider, this treeDataProvider is for control plane, data plane is named as workspaceProvider

Reply:
So, L20-L22 is not needed?

Reply:
ok